com.github.bluesabre.darkbar.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.github.bluesabre.darkbar
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: com.github.bluesabre.darkbar
+finish-args:
+  - '--filesystem=host:ro'
+  - '--filesystem=xdg-config/autostart:create'
```

Comment:
It probably doesn't particularly matter, given how many permissions the app already has, but autostart should be ideally handled with the background portal: https://flatpak.github.io/xdg-desktop-portal/#gdbus-method-org-freedesktop-portal-Background.RequestBackground